students/kojalowicz_piotr/lesson_01/lesson_01.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+import math
+# ===================================
+# Lesson 1. Input, print and numbers
+# ===================================
+
+# Input, print and numbers
+
+a = int(input())
+
+b = int(input())
+
+c = int(input())
+
+print(a + b + c)
+
+# ==========================
+
+# Area of right-angled triangle
+
+b = int(input())
+
+h = int(input())
+
+print((b*h)/2)
+
+# ==========================
+
+# Hello, Harry!
+
+a = input()
+
+
+print("Hello, "+a+"!")
+
+# =====================================
+
+# Apple sharing
+
+n = int(input())
+
+k = int(input())
+
+
+print(k // n)
+
+print(k % n)
+
+# ==========================
+
+# Previous and next
+
+a = int(input())
+
+print("The next number for the number %s is %s."% (a, (a+1)))
+
+print("The previous number for the number %s is %s."% (a, (a-1)))
+
+
+# =============================================
+
+# School desks
+
+a = int(input())
+
+b = int(input())
+
+c = int(input())
+
+
+d= a//2+b//2+c//2 + a%2+b%2+c%2
+
+print(d)
+
+
+# ===========================
+# Lesson 2. Integer and float numbers
+# ===========================
+
+# Last digit of integer
+
+
+a = int(input())
+
+print(a%10)
+
+# =============================
+
+# Tens digit
+
+
+a = int(input())
+
+
+print((a%100)//10)
+
+# ============================
+
+# Sum of digits
+
+
+a = int(input())
+
+b=a%10 + (a//10)%10  + (a//100)%10 
+
+
+print(b)
+
+# =================================
+
+# Fractional part
+
+
+a = float(input())
+
+
+print(a-(a//1))
+
+# =================================
+
+# First digit after decimal point
+
+
+a = float(input())
+
+
+b = ((a*10)//1)%10
+
+print(b)
+
+
+# ==========================
+
+# Car route
+
+
+N = int(input())
+
+M = int(input())
+
+m = math.ceil(M/N)
```

Comment:
Python to nie matematyka ;). Zmienne rób bardziej opisowe. Lowercasem rozdzielanym underscorami (`_`).
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names